Gemfile

```diff
@@ -7,5 +7,4 @@ group :test do
 end
 
 group :profile do
-  gem 'ruby-prof'
```

Comment:
hmm, ok -- if you do a pull request for our changes back to the original project, I'd exclude this one